### ISSUE TITLE
Fast-forward desync involving Badeline orbs

### ISSUE THREAD

**Issue by Kataiser** [ShootMe/CelesteTAS#45]
(Everest v1312) Noticed while TASing Nameguy's 6D that fast-forwarding would kill Madeline, but normal playback wouldn't. Narrowed it down to line 789 of [6D.tas](https://github.com/ShootMe/CelesteTAS/files/4081320/6D.tas.txt), moving that breakpoint one line up or removing it prevents a death. Seems that Badeline orbs get spawned at a different time or spot.

Here's a comparison gif between the two breakpoint locations: 
![comparison](https://user-images.githubusercontent.com/14226721/72668681-03af2b80-39ef-11ea-8f97-2ced5fca9cc9.gif)

**Comment by EuniverseCat** [comment]
We no longer use the code here, bugs should be reported for CelesteTAS-EverestInterop. This is fixed in maingame, i'll try to figure out why it breaks in custom maps.

**Comment by Kataiser** [comment]
Yeah I realized that some time after I created this issue but didn't care enough to create a new one
